### PR TITLE
feat: add priority field to conversation

### PR DESCRIPTION
**Pull request by scmmishra**
Seems like this can be done with minimal impact, tried it on staging instance with 2M rows, happened within 200ms

## References

- [StackOverflow](https://stackoverflow.com/a/19527223 )
- [PostgreSQL Docs](https://www.postgresql.org/docs/current/sql-altertable.html#:~:text=When%20a%20column,the%20table%20required.)

> When a column is added with ADD COLUMN and a non-volatile DEFAULT is specified, the default is evaluated at the time of the statement and the result stored in the table's metadata. That value will be used for the column for all existing rows. If no DEFAULT is specified, NULL is used. In neither case is a rewrite of the table required.

### CODE REVIEW

**Comment by scmmishra** [comment]
@sojan-official raised this PR against a separate branch

**Review by sojan-official — APPROVED** [review]

File: db/schema.rb [hunk 1]
```diff
@@ -437,7 +437,7 @@
     t.datetime "agent_last_seen_at"
     t.jsonb "additional_attributes", default: {}
     t.bigint "contact_inbox_id"
-    t.uuid "uuid", default: -> { "public.gen_random_uuid()" }, null: false
+    t.uuid "uuid", default: -> { "gen_random_uuid()" }, null: false
```
Comment (pranavrajs) [inline db/schema.rb:440]: Is this intentional?

**Comment by pranavrajs** [comment]
@scmmishra Any reason why this is merged on a branch?

**Comment by scmmishra** [comment]
> @scmmishra Any reason why this is merged on a branch?

Wanted to merge the API side also before merging it to develop

**Comment by pranavrajs** [comment]
@scmmishra Let us only do feature branches if it is absolutely required. In this case, having the change in develop doesn't hurt.

**Comment by scmmishra** [comment]
@pranavrajs noted, makes sense. CC: @muhsin-k

**Pull request merged**